client/app/scripts/superdesk-archive/module.js

```diff
@@ -57,9 +57,15 @@ define([
             this.count = 0;
         };
 
+        this.items = getItems;
```

Comment:
yep exactly. but it's kind of weird when mixed together with `this.reset = function () { ... };`
will clean this a bit